pkgs/by-name/pp/ppsspp/package.nix

```diff
@@ -1,25 +1,29 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, SDL2
-, cmake
-, copyDesktopItems
-, ffmpeg_4
-, glew
-, libffi
-, libsForQt5
-, libzip
-, makeDesktopItem
-, makeWrapper
-, pkg-config
-, python3
-, snappy
-, vulkan-loader
-, wayland
-, zlib
-, enableQt ? false
-, enableVulkan ? true
-, forceWayland ? false
+{
+  lib,
+  SDL2,
+  cmake,
+  copyDesktopItems,
+  fetchFromGitHub,
+  ffmpeg,
+  glew,
+  libffi,
+  libsForQt5,
+  libzip,
+  makeDesktopItem,
+  makeWrapper,
+  pkg-config,
+  python3,
+  snappy,
+  stdenv,
+  vulkan-loader,
+  wayland,
+  zlib,
+
+  enableQt ? false,
+  enableVulkan ? true,
+  forceWayland ? false,
+  useSystemFfmpeg? false,
```

Comment:
I know that we generally prefer to use system libraries, but I would set this to `true` as default considering that the non-bundled `ffmpeg` is causing issues.

Reply:
I am not understanding your point.

`useSystemFfmpeg`, when set as true, picks the FFMPEG from system - namely, the Nixpkgs one.
When set as false, it picks the one provided by PPSSPP source code.

Reply:
Oh ok sorry, I was sure this flag was set to default.